### ISSUE TITLE
[spaced-comment] Support typescript triple slash comments

### ISSUE THREAD

**Issue by StephenLarkin** [typescript-eslint/typescript-eslint#600]
The default eslint rule spaced-comment does not support the triple-slash method of typescript file referencing. If it is turned on it will add a space between the second and third slash, breaking the reference.

**Repro**

```JSON
{
  "rules": {
    "spaced-comment": ["error", "always"],
  }
}
```

```TS
/// <reference path="yourFile.ts" />

//A comment
```

**Expected Result**

```TS
/// <reference path="yourFile.ts" />

// A comment
```

**Actual Result**

```TS
// / <reference path="yourFile.ts" />

// A comment
```

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `ESLint`                           | `5.16.0` |

**Comment by kaicataldo** [comment]
Have you tried using `spaced-comment`'s [`markers`](https://eslint.org/docs/rules/spaced-comment#markers) configuration option. The following is considered correct by ESLint and won't warn:

```js
/* eslint spaced-comment: ["error", "always", { "markers": ["/"] }] */

/// This is a comment with a marker
```

**Comment by StephenLarkin** [comment]
@kaicataldo I have not, that's actually perfect. Sorry, I should have double checked the rule, I'm doing a migration from tslint and my life has become rule doc overload.
